src/views/News/components/NewsList/newsList.scss

```diff
@@ -1,12 +1,15 @@
 @import '../../../../vars';
 
+// Leaving for now as the aesthetic of the news page takes shape
```

Comment:
Good idea. As we have more and more UI things showing we can start getting feedback about what looks best and go over all the style pieces again.